src/plugins/telemetry/server/telemetry_collection/get_data_telemetry/constants.ts

```diff
@@ -51,18 +51,18 @@ export const DATA_DATASETS_INDEX_PATTERNS = [
   { pattern: 'fluentbit*', patternName: 'fluentbit' },
   { pattern: '*nginx*', patternName: 'nginx' },
   { pattern: '*apache*', patternName: 'apache' }, // Already in Security (keeping it in here for documentation)
-  // { pattern: '*logs*', patternName: 'third-party-logs' }, Disabled for now
+  { pattern: '*logs*', patternName: 'third-party-logs' },
 
   // Security - Elastic
-  { pattern: 'logstash-*', patternName: 'logstash', shipper: 'logstash' },
+  { pattern: '*logstash*', patternName: 'logstash', shipper: 'logstash' },
```

Comment:
why did you not also adjust line 41, 46?

Reply:
@ruflin  Forgot to push the latest change, my bad - please have another look